admin/class-asset.php

```diff
@@ -169,7 +168,7 @@ public function __construct( array $args ) {
 		$this->name      = $args['name'];
 		$this->src       = $args['src'];
 		$this->deps      = $args['deps'];
-		$this->version   = $args['version'];
+		$this->version   = isset( $args['version'] ) ? $args['version'] : '';
```

Comment:
If we move the `version` and `suffix` to the `defaults` with empty values; we don't have to do any conditionals here because of the `array_merge`.